sos/plugins/openstack_cinder.py

```diff
@@ -27,8 +27,10 @@ class OpenStackCinder(Plugin):
     plugin_name = "openstack_cinder"
     profiles = ('openstack', 'openstack_controller')
 
-    option_list = [("db", "gathers openstack cinder db version", "slow",
-                    False)]
+    option_list = [
+        ("db", "gathers openstack cinder db version", "slow", False),
+        ("exclude", "Ignore files matching this path spec", "", "")
+    ]
```

Comment:
From the information you have given so far:

- Add a (possibly gobal) option "json" or similar, to request collection of (possibly duplicate) data in JSON format. If global then plugins can set their own default for this, but if the option is given on the command line, then all plugins are affected.

- If the option is absent and the plugin author chooses to not collect some JSON data by default then either forbid (less preferable), or avoid adding (preferable) the corresponding files.

- From the example above the pattern'd be something like "*-json.log"

- If JSON file patterns or locations need to be set on a per-plugin basis that's also possible but may require introducing a new interface to mark the data separately from configuration/log data.